docs/faq/CodeStructure.md

```diff
@@ -72,3 +73,27 @@ Find the balance between these two extremes, and you will master Redux.
 - [#1165: Where to put business logic / validation?](https://github.com/reactjs/redux/issues/1165)
 - [#1171: Recommendations for best practices regarding action-creators, reducers, and selectors](https://github.com/reactjs/redux/issues/1171)
 - [Stack Overflow: Accessing Redux state in an action creator?](http://stackoverflow.com/questions/35667249/accessing-redux-state-in-an-action-creator/35674575)
+
+
+<a id="structure-action-creators"></a>
+### Why should I use action creators?
+
+Action creators are superior to inline actions for several reasons. 
```

Comment:
Thanks for the review! I agree - 'superior to inline actions' has an imperious tone I didn't intend.